exercises/concept/high-school-sweetheart/.docs/introduction.md

```diff
@@ -42,11 +42,17 @@ Elixir provides many functions for working with strings in the `String` module.
 
 ## Pipe Operator
 
-The `|>` operator is called the pipe operator. It can be used to chain function calls together in such a way that the value returned by the previous function call is passed as the first argument to the next function call.
+The `|>` operator is called the pipe operator. It is used to chain function calls together in such a way that the value returned by the previous function call is passed as the first argument to the next function call.
+
+When using the pipeline, you can place the `|>` operator after the result of a function call to pass its return value as the first argument to the next function. However, there is no need to use the `|>` operator on the first function call, as you can simply call the function with its arguments.
```

Comment:
This seems to conflict with [pipe-operator/about.md](https://github.com/exercism/elixir/blob/main/concepts/pipe-operator/about.md), which notes pipelines such as `String.upcase("hello") |> String.split("")` go against best practice.

It could also cause confusion as we do the opposite in the example below.

Reply:
Thank you for your review and feedback! I appreciate your attention to detail. You are correct that there may be some confusion in the current description of the pipe operator. I addressed this issue by updating the introduction to make it clearer and more consistent with the best practices.

Let me know if you have any other suggestions or concerns :).